txstratum/utils.py

```diff
@@ -317,7 +317,7 @@ def calculate_expected_mining_time(miners_hashrate_ghs: float, job_weight: float
       5  | 0.9932
     """
     if miners_hashrate_ghs == 0:
-        return -1
+        return 5
```

Comment:
Again just think we should add a comment here explaining the reason behind this change. Maybe use part of the PR description.

Reply:
Done. Can you review again, please?

Reply:
It's great!